Kuzzle.Tests/API/Controllers/AdminControllerTest.cs

```diff
@@ -0,0 +1,245 @@
+﻿using Xunit;
+using KuzzleSdk.API.Controllers;
+using Newtonsoft.Json.Linq;
+using KuzzleSdk.Utils;
+
+namespace Kuzzle.Tests.API.Controllers {
+  public class AdminControllerTest {
+    private readonly AdminController _adminController;
+    private readonly KuzzleApiMock _api;
+
+    public AdminControllerTest() {
+      _api = new KuzzleApiMock();
+      _adminController = new AdminController(_api.MockedObject);
+    }
+
+
+    [Theory]
+    [InlineData(false)]
+    [InlineData(true)]
+    public async void DumpAsyncTestSuccess(bool value) {
```

Comment:
Useless `acknowledge` value theory: you forgot to change that when dropping the handling of the "acknowledge" response boolean value.
(this comment also applies to other tests below)

Reply:
I've removed the forgotten variables that i've used.